src/Views/login.php

```diff
@@ -6,11 +7,11 @@
     <head>
         <meta charset="UTF-8">
         <title>Example Login Form</title>
-        <link href="style.css" rel="stylesheet">
+        <link href="http://local.dev/Project1/public/style.css" rel="stylesheet">
```

Comment:
Change the hardcoded environment server to a php variable for the current server running the app eg. $_SERVER["SERVER_NAME"]